yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/create/SessionSpec.java

```diff
@@ -439,6 +465,7 @@ private static final class Property {
         private static final String IDENTITY_PROFILE_REQUIREMENTS = "identity_profile_requirements";
         private static final String SUBJECT = "subject";
         private static final String RESOURCES = "resources";
+        private static final String IDENTITY_PROFILE_PREVIEW = "create_identity_profile_preview";
```

Comment:
Can you set the property name to match the string please?

Reply:
Done